portal/src/locale-data/en.json

```diff
@@ -106,9 +106,12 @@
   "UsersList.column.last-login-at": "Last Login",
 
   "UsersList.disable-user": "Disable User",
+  "UsersList.enable-user": "Enable User",
 
-  "DisableUserDialog.title": "Disable User",
-  "DisableUserDialog.text": "Do you really want to disable user {username}",
+  "SetUserDisabledDialog.disableUser.title": "Disable User",
+  "SetUserDisabledDialog.disableUser.text": "Do you really want to disable user {username}?",
```

Comment:
The description here would be more meaningful if we mention the consequence here.

> Disabling the user {username} will forbid them from logging in. Their existing sessions will also become invalid.

And the buttons should be a secondary button "Cancel" and a destructive button "Disable"